CMakeLists.txt

```diff
@@ -406,3 +406,9 @@ ADD_CUSTOM_TARGET(release
 
 feature_summary(WHAT ALL
                 DESCRIPTION "=== TiledArray Package/Feature Info ===")
+
+option(TA_PYTHON "Build TA python module" ON)
```

Comment:
should build this by default only if `BUILD_SHARED_LIBS=ON`? see `redefaultable_option`

Reply:
I m actually going to make it off by default as it could create some issues in mpqc build